sound/soc/soc-topology.c

```diff
@@ -906,9 +909,6 @@ static int soc_tplg_dmixer_create(struct soc_tplg *tplg, unsigned int count,
 			continue;
```

Comment:
paranoia check: what happens if the control creation fails, would there be a need to delete the tlv here? same for the change below as well.

Reply:
@plbossart Thanks. I missed it.